### ISSUE TITLE
[Bug] No notification/lockscreen media controls

### ISSUE THREAD

**Issue by Roonil** [YTVanced/Vanced#1697]
**Bug description:**
The notification controls worked fine uptil version 16.16.38. After that all I get is a blank youtube notification which just has the title of the video. But if I tap on the area where the controls are supposed to be present, they do work but they do not show up. BUT if I use the light theme on my phone then I'm able to see the controls, but still I can't see the thumbnail in the notification area. It used to work fine before. Maybe it's a YouTube issue rather than a Vanced one? And I even tried color inversion while in dark mode on my device to see if it was some sort of a theming bug, but it still doesn't show the controls. I've tried toggling all sorts of notifications and even allowing battery optimisation to skip YouTube, but still no luck.
Also, there are NO notification managers running or anything that requires notification access that might conflict with Vanced, but even then the previous versions work fine.

**Have you read vanced guides already? (YES/No)**
YES 

**Variant:**
Root

**Vanced version:**
17.03.38

**Link used to download vanced:**
https://vancedapp.com/

**Android version:**
Android 9.0

**Device:**
Asus ROG Phone I

**Steps to Reproduce**
Steps to reproduce the error:
1. Open the app
2. Play any video
3. Go to the home screen
4. Open notifications, the controls are supposed to be there but they are not

**Expected behaviour:**
The notification controls including the thumbnail should be visible.

The bug (device is in dark mode, no controls visible):
![Screenshot_20220228-234423311](https://user-images.githubusercontent.com/15421185/156036339-6fa41323-fb11-4399-bd55-a954dda6bbee.jpg)

Now the device is following the light theme, but still no thumbnail shows up:
![Screenshot_20220228-234506149](https://user-images.githubusercontent.com/15421185/156036414-876a5a25-4107-464a-a4ae-7a3a3220c5a3.jpg)

**Comment by RealKasumi** [comment]
thumbnail no longer shows up because of a stock update, blame google for that
as for the controls, it's most likely down to your rom

**Comment by Roonil** [comment]
I see. And even I thought it was something related to my device, how can I exactly track the issue then? Btw I'm using the stock Android Pie version on my phone.

**Comment by Roonil** [comment]
UPDATE: I tried something just now, I removed the latest Vanced version and then proceeded to download the last working version (16.16.38) and confirmed the notifications were working. Then I updated the Vanced versions, one-by-one, and this time the notifications were working INCLUDING the thumbnail.... What's going on?

This screenshot is on the latest 17.03.38 after I sequentially applied the updates.

![Screenshot_20220301-000048470](https://user-images.githubusercontent.com/15421185/156038988-fa770e5c-1e18-476b-be39-ea21ebb29ee9.jpg)
